src/style/app.css

```diff
@@ -1,584 +1,759 @@
+
   font-size: calc(2vh + 2vw);
   text-align: center;
   color: #ffffff;
+
   background-color: #b1d4df;
-  padding: calc(1.5vh + 1.5vw); }
+  border-radius: calc(1vh + 1vw);
+}
 
 .game-btn {
-  text-decoration: none; }
+  text-decoration: none;
+}
 
 .game-wrapper_menu-page {
   justify-content: center;
-  text-align: center; }
+
+  text-align: center;
+}
 
 .game-menu {
+  padding: 0;
+
   list-style-type: none;
-  padding: 0; }
+}
 
 .menu-option {
-  margin-top: 5vh; }
+  margin-top: 5vh;
+}
 
 .menu-link {
-  text-decoration: none;
-  color: #aed4dc;
+  margin-bottom: 3vh;
+
   text-align: center;
-  margin-bottom: 3vh; }
+  color: #aed4dc;
+  text-decoration: none;
+}
 
 .menu-option:first-child {
-  margin: 0; }
+  margin: 0;
+}
 
 .header {
-  width: 100%;
-  padding: 4vh 0;
-  text-align: right;
   display: flex;
   flex-flow: row nowrap;
   justify-content: space-between;
+  width: 100%;
+  padding: 4vh 0;
+
+  text-align: right;
+
+  background-color: #c3e9f1;
   border-bottom: calc(0.3vh + 0.3vw) solid #b1d4df;
-  background-color: #c3e9f1; }
+}
 
 .header-empty {
-  justify-content: flex-end; }
+  justify-content: flex-end;
+}
 
 .close-btn {
   display: block;
-  margin-right: 6vw;
+  box-sizing: border-box;
   width: calc(4vw + 4vh);
   height: calc(4vw + 4vh);
+  margin-right: 6vw;
+
   background-color: #8abfc6;
   background-image: url("../img/close-btn.png");
-  background-size: 60%;
+  background-repeat: no-repeat;
   background-position: 50% 50%;
+  background-size: 60%;
   border: calc(0.25vw + 0.25vh) solid #5e9aa4;
-  background-repeat: no-repeat;
   border-radius: calc(0.5vw + 0.5vh);
-  box-sizing: border-box; }
+}
 
 .question-answer-wrapper {
-  font-size: calc(2vh + 2vw);
-  color: #83b4c1;
   display: flex;
   flex-flow: column;
   justify-content: center;
   align-items: center;
+  width: 100%;
   height: 100%;
-  width: 100%; }
+
+  font-size: calc(2vh + 2vw);
+  color: #83b4c1;
+}
 
 .question-field-wrapper {
   display: flex;
   flex-direction: column;
   align-items: center;
-  height: 100%;
   width: 95%;
-  margin-top: 5vh; }
+  height: 100%;
+  margin-top: 5vh;
+}
 
 .question-title {
-  color: #ca8e6c;
-  font-size: calc(2.3vh + 2.3vw);
+  margin-bottom: 1.5vh;
+
   font-weight: bold;
-  margin-bottom: 1.5vh; }
+  font-size: calc(2.3vh + 2.3vw);
+  color: #ca8e6c;
+}
 
 .question-text {
-  color: #000000;
   width: 90%;
-  text-align: center; }
+
+  text-align: center;
+  color: #000000;
+}
 
 .answer-field-wrapper {
   display: flex;
   flex-flow: row nowrap;
   justify-content: space-between;
   align-items: center;
+  width: 100%;
   height: 10vh;
+
   border-top: calc(0.3vh + 0.3vw) solid #b1d4df;
-  width: 100%; }
+}
 
 .answer-time-count {
+  bottom: 50%;
+  left: 8vw;
+
   width: 12vw;
   height: 12vw;
-  left: 8vw;
-  bottom: 50%;
+  margin-left: 4vw;
+
+  font-size: calc(2vh + 2vw);
+  line-height: 12vw;
+  text-align: center;
+
   background-color: rgba(255, 235, 59, 0.7);
```

Comment:
Here is the same. But only before

Reply:
It happened because all properties were divided into logical groups (position, block-model, font and so on). each group has a space line before and after itself.

Reply:
@JuliaMV 
Thank you for your answer.
It's clear.
And this is not a problem. Only I had never seen such a division of the new line of groups properties in the a single block.
It was just a question. And if we accept this behavior in this project and our codestyle then everything is fine!

**FLY: @davojta, @AntiHero  **

Reply:
We can add this information in the part "Formatting Code Automatically".
Divide all css-rules into logical groups is recommended HTML Academy styleguide but I can remove these line breaks if they take a lot of space or for other reasons ;)